scripts/camViewer

```diff
@@ -202,30 +202,39 @@ NUDGE=0
 
 OPTIONS=$(getopt -o lmredasnc:w:u:H: --long list,main,reboot,enable,disable,acquire,stop,nudge,cam:,rate:,hutch: -n \'$0\' -- "$@")
 
-if [ $? != 0 ] ; then echo "Terminating..." >&2 ; exit 1 ; fi
-
-
+if [ $? != 0 ]; then echo "Terminating..." >&2 ; exit 1 ; fi
 
 eval set -- "$OPTIONS"
 
-
-
 while [ : ]
 do
     case "$1" in
-        -l|--list)  
+        -l|--list)
             LIST=1
             shift
             ;;
         -c|--cam)
             CAM=$2
-            shift 2 ;;
+            shift 2
+            ;;
         -u|--rate)
             RATE=$2
-            shift 2 ;;
+            shift 2
+            ;;
         -H|--hutch)
             hutch=$2
-            shift 2 ;;
+            echo "HUTCH: $hutch" >&2
+            #  If *not* already running in an SSH session, SSH to kfe-console
+            if [[ -z "$IS_REMOTE_SSH" ]] && [ "$hutch" == "kfe" ]; then
+                echo "Not in SSH, connecting to kfe-console..." >&2
+                # Pass all arguments *before* IS_REMOTE_SSH=1
+                ssh -t kfe-console "IS_REMOTE_SSH=1 $DIR/$(basename "$0") \"$@\" ; bash"
```

Comment:
Is there any risk of command-line parameters getting dropped from `$@` thanks to the `shift` calls in the rest of the cli handling here? I suspect we need to cache the original args and pass them unaffected by `shift`.